common-rest/src/main/java/org/uniprot/api/common/repository/search/QueryResult.java

```diff
@@ -16,71 +17,16 @@
  * @author lgonzales
  */
 @Getter
+@Builder
 public class QueryResult<T> {
     private final Collection<TermInfo> matchedFields;
     private final Collection<Suggestion> suggestions;
     private Page page;
     private final Stream<T> content;
     private final Collection<Facet> facets;
-    private final Collection<String> failedIds;
+    private final IdMappingStatistics idMappingStatistics;
     private final Collection<ProblemPair> warnings;
```

Comment:
I guess, we are using failedIds in other apis also not just idmapping. If thats the case we should rename the class IdMappingStatistics

Reply:
What name do you Suggest?